File: json/json-annotation-processor/src/test/java/io/koraframework/json/annotation/processor/dto/DtoWithJsonSkip.java
package io.koraframework.json.annotation.processor.dto;

import io.koraframework.json.common.annotation.Json;
import io.koraframework.json.common.annotation.JsonSkip;

@Json
public record DtoWithJsonSkip(String field1, String field2, @JsonSkip String field3, @JsonSkip String field4) {

}

File: json/json-symbol-processor/src/test/kotlin/io/koraframework/json/ksp/dto/DtoWithJsonSkip.kt
package io.koraframework.json.ksp.dto

import io.koraframework.json.common.annotation.Json
import io.koraframework.json.common.annotation.JsonSkip

@Json
data class DtoWithJsonSkip(
    val field1: String,
    val field2: String,
    @JsonSkip val field3: String,
    @JsonSkip val field4: String
)
